lib/matplotlib/rcsetup.py

```diff
@@ -843,7 +843,7 @@ def validate_hist_bins(s):
     ## patch props
     'patch.linewidth':   [None, validate_float_or_None],  # line width in points
     'patch.edgecolor':   ['k', validate_color],  # black
-    'patch.facecolor':   ['b', validate_color],  # blue
+    'patch.facecolor':   ['#1f77b4', validate_color],  # blue
```

Comment:
It's a blue, just a different one ;)   But I agree, the comment should probably be updated.